### PR TITLE
add debt description

### LINKED ISSUE #63: Refactor the code to allow debt to be created with a description
Necessary for US06

### LINKED ISSUE #72: Add debt desciption to the UI
Add debt description text field to the UI

### PR DESCRIPTION
**Pull request by yenanw**
- resolves #63
- resolves #72

### CODE REVIEW

**Comment by yenanw** [comment]
still working in progress, will be open once debt description is added to both GUI and model

**Review by OlofSjogren — APPROVED** [review]
Looks good, clean additions.

**Comment by yenanw** [comment]
nvm, found an error

**Review by Puh00 — COMMENTED** [review]
Looks good to me!
image

File: app/src/main/java/com/goayo/debtify/model/ModelEngine.java [hunk 1]
```diff
@@ -50,21 +48,21 @@ private void HARDCODEDDEBTDATA() {
         Set<String> debtData1 =  new HashSet<>();
         debtData1.add("0735216752");
         try {
-            createDebt("1003", "0756415987", debtData1, 23.90);
+            createDebt("1003", "0756415987", debtData1, 23.90, "ass");
```
Comment (gabrielbrattgard) [inline app/src/main/java/com/goayo/debtify/model/ModelEngine.java:51]: offensive.

**Review by gabrielbrattgard — APPROVED** [review]

**Review by OscarSanner — APPROVED** [review]

**Pull request merged**